src/XMakeTasks/CultureInfoCache.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Framework.Globalization
+{
+    /// <summary>
+    /// Provides read-only cached instances of <see cref="CultureInfo"/>.
+    /// <remarks>
+    /// Source: https://raw.githubusercontent.com/aspnet/Localization/dev/src/Microsoft.Framework.Globalization.CultureInfoCache/CultureInfoCache.cs
```

Comment:
Can you specify the commit hash you're pulling from in the commit message?

(I'll spare you my super-nitpicky "bring this in by itself via `--orphan`, then merge it in" nonsense :smile:)